pkg/spiffe/spiffe.go

```diff
@@ -52,7 +52,7 @@ func GenSpiffeURI(ns, serviceAccount string) (string, error) {
 func MustGenSpiffeURI(ns, serviceAccount string) string {
 	uri, err := GenSpiffeURI(ns, serviceAccount)
 	if err != nil {
-		log.Error(err.Error())
+		log.Warn(err.Error())
```

Comment:
this should be debug.. not a warning.. most people have their log levels set to info

Reply:
I dropped the level down